### ISSUE TITLE
Pressing `space` on a Select scrolls the page down and moves select dropdown away from element

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13252]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Related to comment https://github.com/rancher/dashboard/issues/12775#issuecomment-2623139251, 

When hit inside dropdowns, the space bar doesn't just select the element but also scrolls the whole page down, as seen in the screenshots. The ideal behavior should only be pressing the space bar to select the element.

https://github.com/user-attachments/assets/cc26d287-dee7-487c-add8-422223986562
https://github.com/user-attachments/assets/6477da53-86b1-43ad-ad01-b014e2eb6b6b

https://github.com/user-attachments/assets/856fc63e-1337-4867-8a90-231126e653e9

This should be fixed in `Select` and `LabeledSelect` at least.

**To Reproduce**

Go to the `User preferences page`, go to a `Select` component (focused with keyboard nav) and hit `space` to open, or open it and hit `space` to select one of the options.
Also identified in the charts page

**Result**
Select dropdown is open/option is selected but the page scrolls

**Expected Result**

page shouldn't scroll while interacting with these UI elements

**Screenshots**

**Additional context**

**Comment by IsaSih** [comment]
Might be related https://github.com/rancher/dashboard/issues/13427

**Comment by izaac** [comment]
Firefox version: `135.0.1`
Chrome version: `133.0.6943.127`

Firefox this is working with:

- Enter ✅ 
- Space ✅ 
- Arrow Down ✅ 

On Chrome however the `Arrow Down` is making a slight scroll of the dropdown options away from the dropdown input field.

https://github.com/user-attachments/assets/9407b242-e2f3-4bf5-bbe4-139d87d6aa2e

**Comment by izaac** [comment]
Rancher version: `v2.11-13170b249a26f304b1ae89a96688f547a96b77b1-head`
UI version: `master 28b6f80`

Chrome version: `133.0.6943.141`

The remaining issue with Chrome is now fixed ✅ . Closing
